aws-api-appsync/src/main/java/com/amplifyframework/core/model/types/JavaFieldType.java

```diff
@@ -28,86 +30,83 @@
     /**
      * Represents the boolean data type.
      */
-    BOOLEAN(Boolean.class.getSimpleName()),
+    BOOLEAN(Boolean.class),
 
     /**
      * Represents the int data type.
      */
-    INTEGER(Integer.class.getSimpleName()),
+    INTEGER(Integer.class),
 
     /**
      * Represents the long data type.
      */
-    LONG(Long.class.getSimpleName()),
+    LONG(Long.class),
 
     /**
      * Represents the float data type.
      */
-    FLOAT(Float.class.getSimpleName()),
+    FLOAT(Float.class),
 
     /**
      * Represents the String data type.
      */
-    STRING(String.class.getSimpleName()),
+    STRING(String.class),
+
+    /**
+     * Represents the java.lang.Date data type.
+     */
+    JAVA_DATE(Date.class),
 
     /**
      * Represents the Date data type.
      */
-    DATE(Temporal.Date.class.getSimpleName()),
+    DATE(Temporal.Date.class),
 
     /**
      * Represents the DateTime data type.
      */
-    DATE_TIME(Temporal.DateTime.class.getSimpleName()),
+    DATE_TIME(Temporal.DateTime.class),
 
     /**
      * Represents the Time data type.
      */
-    TIME(Temporal.Time.class.getSimpleName()),
+    TIME(Temporal.Time.class),
 
     /**
      * Represents the Timestamp data type.
      */
-    TIMESTAMP(Temporal.Timestamp.class.getSimpleName()),
+    TIMESTAMP(Temporal.Timestamp.class),
     
     /**
      * Represents the Enum type.
      */
-    ENUM(Enum.class.getSimpleName()),
+    ENUM(Enum.class),
 
     /**
      * Represents the Model type.
      */
-    MODEL(Model.class.getSimpleName()),
+    MODEL(Model.class),
 
     /**
      * Represents any custom type (objects that are not models).
      */
-    CUSTOM_TYPE(Object.class.getSimpleName());
+    CUSTOM_TYPE(Object.class);
 
-    private final String javaFieldType;
+    private final Class<?> javaFieldType;
 
-    JavaFieldType(@NonNull String javaFieldType) {
+    JavaFieldType(@NonNull Class<?> javaFieldType) {
         this.javaFieldType = javaFieldType;
     }
 
-    /**
-     * Return the string that represents the value of the enumeration constant.
-     * @return the string that represents the value of the enumeration constant.
-     */
-    public String stringValue() {
```

Comment:
This one wasn't being used anywhere, huh?

Reply:
@jamesonwilliams right this one was only used internally inside the ```JavaFieldType.java```